service/s3: Add clarification to bucket notification docs (#10406)
terraform-providers/terraform-provider-aws: website/docs/r/s3_bucket_notification.html.markdown

Manages a S3 Bucket Notification Configuration. For additional information, see the [Configuring S3 Event Notifications section in the Amazon S3 Developer Guide](https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html).

> ~> **NOTE:** S3 Buckets only support a single notification configuration. Declaring multiple `aws_s3_bucket_notification` resources to the same S3 Bucket will cause a perpetual difference in configuration. Check exemple "Trigger multiple Lambda" bellow to understand better.

Reviewer: Hi @SBerda! Thanks for adding this. I spotted a couple of typos, so let me know if this matches what you wanted to say.
```suggestion
~> **NOTE:** S3 Buckets only support a single notification configuration. Declaring multiple `aws_s3_bucket_notification` resources to the same S3 Bucket will cause a perpetual difference in configuration. See the example "Trigger multiple Lambda functions" for an option.
```

Author: Yes perfect, thanks :)
